editors/code/src/util.ts

```diff
@@ -114,15 +113,12 @@ export function isRustEditor(editor: vscode.TextEditor): editor is RustEditor {
 export function isValidExecutable(path: string): boolean {
     log.debug("Checking availability of a binary at", path);
 
-    if (!fs.existsSync(path)) return false;
```

Comment:
Right, I had some doubts about that, as, as a run of thumb, I try to never ask about stuff before doing stuff (that's prone to TOCTOU). PATH handling is an issue specific to execution. 

bors r+